fk/secretreceive.go

```diff
@@ -194,6 +198,10 @@ func decryptAPISecret(
 		UUID:             uuid,
 	}
 
+	if !literalData.ForEyesOnly() {
+		decryptedSecret.filename = literalData.FileName
```

Comment:
So what happens if I encode a secret with the filename set to a full path like `/Users/ian/.zsh` ?

Reply:
See https://github.com/fluidkeys/fluidkeys/pull/294/commits/f5b10cbf602c79613655a6187d5d4d4020bc357d